pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -597,15 +614,22 @@ func (dm *DockerManager) runContainer(
 		cpuShares = milliCPUToShares(cpuRequest.MilliValue())
 	}
 	var devices []dockercontainer.DeviceMapping
+	var nvidiaGPUPaths []string
+	nvidiaGPUPlugin := dm.GetNvidiaGPUPlugin()
 	if nvidiaGPULimit.Value() != 0 {
```

Comment:
Just check the predicates, I think so far it's good:

predicates.go:

in func ```go func PodFitsResources(pod *api.Pod, meta interface{}, nodeInfo *schedulercache.NodeInfo) (bool, error) ```
```go
if allocatable.NvidiaGPU < podRequest.NvidiaGPU+nodeInfo.RequestedResource().NvidiaGPU {
		return false,
			newInsufficientResourceError(nvidiaGpuResourceName, podRequest.NvidiaGPU, nodeInfo.RequestedResource().NvidiaGPU, allocatable.NvidiaGPU)
	}
```